packages/modules/itwin-viewer-react/src/services/Initializer.ts

```diff
@@ -203,6 +215,7 @@ class Initializer {
         }
         console.log("iModel.js initialized");
 
+        this._initializing = false;
```

Comment:
Not sure I get why a boolean was needed here instead of the Promise, `initialize()`. Is this the issue we talked about with part of the API not being async?

Reply:
When there are quick re-renders of the component (props being initialized), this would occasionally execute twice before the first promise resolved. This is mostly a fail safe